app/src/main/java/com/anikinkirill/tinkoffsiriusmobile/SignInActivity.java

```diff
@@ -0,0 +1,46 @@
+package com.anikinkirill.tinkoffsiriusmobile;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import java.io.File;
+import java.io.FileInputStream;
+
+public class SignInActivity extends Activity {
```

Comment:
I think AppCompatActivity better then Activity, cause if we use some new features of Android (ToolBars, ActionBars and so on), we extend from AppCompatActivity